.gitignore

```diff
@@ -7,3 +7,4 @@
 # Build output
 bin/
 obj/
+*.user
```

Comment:
❔  What was not caught before?  I expecting the `*.csproj.user` line above to catch these.

Reply:
It was to exclude a file that NCrunch auto-generates. `blar.blar.ncrunch.user` or something. I guess NCrunch assumes a blanket `*.user` exclude.